### DISCUSSION TITLE
ScriptReference/Camera.ScreenToWorldPoint

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#85] [Notes]
# ScriptReference/Camera.ScreenToWorldPoint

https://docs.unity3d.com/ScriptReference/Camera.ScreenToWorldPoint.html

**Comment by vertxxyz** [comment]
Providing distance is important when using a perspective camera, as `0` distance from a camera is the world space position of the camera itself.

When the result is on a fixed plane in space, use a [`Plane`](https://docs.unity3d.com/ScriptReference/Plane.html) and [`ScreenPointToRay`](https://docs.unity3d.com/ScriptReference/Camera.ScreenPointToRay.html) to avoid calculating an appropriate distance from the camera.

(See [Unity Help: ScreenToWorldPoint](https://help.vertx.xyz/programming/specifics/screentoworldpoint)).

**Comment by docscomments** [comment]
**Comment by `@simondarksidej`:**  

Input.mousePosition isn't inverted, unlike Event.mousePosition in the example.
